mlflow_faculty/mlflow_converters.py

```diff
@@ -36,14 +36,22 @@
 from mlflow_faculty.py23 import to_timestamp
 from mlflow.exceptions import MlflowException
 
-_RUN_STATUS_MAP = {
+FACULTY_TO_MLFLOW_RUN_STATUS_MAP = {
```

Comment:
The names should either all be consistently private or consistently not. I vote for making them consistently private.